<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkAddRemoveComboBox</class>
 <widget class="QWidget" name="ctkAddRemoveComboBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select items</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="ComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="AddPushButton">
     <property name="toolTip">
      <string>Add an object</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../CTKWidgets.qrc">
       <normaloff>:/Icons/plus.png</normaloff>:/Icons/plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="RemovePushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Remove select object</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../CTKWidgets.qrc">
       <normaloff>:/Icons/minus.png</normaloff>:/Icons/minus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="EditPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Edit selected object</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../CTKWidgets.qrc">
       <normaloff>:/Icons/edit.png</normaloff>:/Icons/edit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../CTKWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
